package.json

```diff
@@ -15,6 +15,7 @@
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
+    "fmt": "prettier --single-quote --trailing-comma all --tab-width 4 --write \"{src/**/*.ts,*.js}\"",
```

Comment:
Missing `--parser typescript`, leading to potentially weird formatting. Also `--trailing-comma all`  will break Node 4-6 support for js files (e.g. doc-postprocess will fail to run under Node 6 today)